build/ci/evergreen.yml

```diff
@@ -2064,48 +2332,93 @@ buildvariants:
       - name: pkg_test_docker_ubuntu2004
       - name: pkg_test_docker_debian9
       - name: pkg_test_docker_debian10
-  - name: release_msi
-    display_name: "Release (go-msi)"
+  - name: release_mongocli_msi
+    display_name: "Release MongoCLI (go-msi)"
     run_on:
       - windows-vsCurrent-small
     expansions:
       go_root: "c:\\golang\\go1.17"
       go_bin: "c:\\golang\\go1.17/bin"
     tasks:
-      - name: release_msi
-  - name: release_github
-    display_name: "Release (goreleaser)"
+      - name: release_mongocli_msi
+  - name: release_mongocli_github
+    display_name: "Release MongoCLI (goreleaser)"
+    run_on:
+      - macos-1014-codesign
+    expansions:
+      go_root: "/opt/golang/go1.17"
+      go_bin: "/opt/golang/go1.17/bin"
+    tasks:
+      - name: release_mongocli
+  - name: atlascli_release_github
+    display_name: "Release AtlasCLI (goreleaser)"
     run_on:
       - macos-1014-codesign
     expansions:
       go_root: "/opt/golang/go1.17"
       go_bin: "/opt/golang/go1.17/bin"
     tasks:
-      - name: release
-  - name: release_publish_42
-    display_name: "Publish yum/apt 4.2"
+      - name: release_atlascli
+  - name: atlascli_release_msi
+    display_name: "Release AtlasCLI (go-msi)"
+    run_on:
+      - windows-vsCurrent-small
+    expansions:
+      go_root: "c:\\golang\\go1.17"
+      go_bin: "c:\\golang\\go1.17/bin"
+    tasks:
+      - name: release_atlascli_msi
+  - name: release_mongocli_publish_42
+    display_name: "Publish MongoCLI yum/apt 4.2"
+    run_on:
+      - rhel70-small
+    expansions:
+      server_version: "4.2.3"
+      notary_key_name: "server-4.2"
+    tasks:
+      - name: .push_mongocli_stable
+  - name: release_mongocli_publish_44
+    display_name: "Publish MongoCLI yum/apt 4.4"
+    run_on:
+      - rhel70-small
+    expansions:
+      server_version: "4.4.3"
+      notary_key_name: "server-4.4"
+    tasks:
+      - name: .push_mongocli_stable
+  - name: release_mongocli_publish_50
+    display_name: "Publish MongoCLI yum/apt 5.0"
+    run_on:
+      - rhel70-small
+    expansions:
+      server_version: "5.0.0"
+      notary_key_name: "server-5.0"
+    tasks:
+      - name: .push_mongocli_stable
+  - name: atlascli_release_publish_42
```

Comment:
I will rename these variants to `release_atlascli_publish_*` once we are ready to release atlascli